adagucserverEC/CCreateLegend.cpp

```diff
@@ -577,6 +579,9 @@ int CCreateLegend::createLegend(CDataSource *dataSource,CDrawImage *legendImage,
     
     
     if(discreteLegendOnInterval){
+      int maxIterations = 250;
```

Comment:
Waarom 250?

Reply:
maxIterations is the maximum number of labels in the legendgraphic, more makes no sense as legend image is quite small 
Sometimes this function tries to render thousands of labels in a few pixels, maxIterations prevents this.